### ISSUE TITLE
No way to get out of error in browser but to refresh the page.

### ISSUE THREAD

**Issue by joelrb** [prisma/studio#1216]
### Bug description

Adding a record but the id I entered does not exist in another table. Error occurs because of relation.

But I can't get out of this error. I have to refresh the page.

![Screenshot 2024-03-05 012102](https://github.com/prisma/studio/assets/225284/ac2aee0b-99f8-4ea5-859f-9b7526533194)

### Expected behavior

Able to close the error without refreshing the page.

### Environment & setup

- Link to Data Browser instance: 
- OS: Windows
- Browser: Edge
- Database: PostgreSQL

### Screenshot of page where the problem happened (if applicable)

![Screenshot 2024-03-05 012102](https://github.com/prisma/studio/assets/225284/10171f31-34f9-43fe-8ca7-b354db46c4dd)

**Comment by janpio** [comment]
Doesn't clicking _next_ to the error close the modal?

**Comment by joelrb** [comment]
> Doesn't clicking _next_ to the error close the modal?

Nope. Nothing happens. Doesn't close clicking anywhere inside and outside the modal. The error is just too long or the modal is very narrow, it should auto set the modal width based on the content.

**Comment by igalklebanov** [comment]
Hey :wave:

Prisma ORM 7 introduces a brand new standalone [Studio](https://www.prisma.io/docs/orm/tools/prisma-studio), built from the ground up to work everywhere, with or without Prisma ORM. Consequently, all "old Studio" issues are being closed as there won't be any further work on it. :mega:

The new Studio is also available in the Prisma VS Code extension and in the Prisma Console application. :electric_plug:

Help shape, improve and fix the new Studio with us in this repository by submitting bug reports and feature requests. :construction_worker:
